### ISSUE TITLE
treatment of fuzzy concepts

### ISSUE THREAD

**Issue by LinguList** [clics/clics2#32]
I was just looking at the graph for [fingernail](http://clics.clld.org/graphs/infomap_62_FINGERNAIL), and I realized that due to concepticon fuzzy concepts like [claw or nail](http://concepticon.clld.org/parameters/2128), we have at times strange networks. 

I don't consider this as a huge problem, but imagine we could automatically expand these and add two translations for the respectively narrower concepts in the hierarchy. As we would still store the "original concept", this might cause less problems in terms of consistency. The only caveat is the depth of going down, so I'd restrict this to one step in the network (arm or hand -> arm / hand, not more).

**Comment by xrotwang** [comment]
I also think that this is not a huge problem - rather the opposite: It validates our approach of keeping the concepts as encountered in the source, while allowing to refine it downstream. You could turn this into an example of why data quality  matters, by going through all networks for "A OR B" concepts and catalog the strangeness this causes.

**Comment by LinguList** [comment]
Yes, in fact, it is difficult to assess, whether they really answered properly in the questionnaire with "arm/hand", so splitting them is probably also just leading to more mud in the data. Maybe it's enough to write some time a little blog post about this, pointing interested users to the fact that they should be careful, and also think of the hierarchies in concepticon. In addition, we could print out our currently used concepticon hierarchy in clics and point to it in future versions of the application. In fact, this is probably an example for a howto-blog / cookbook, as it would illustrate how to check concepticon's hierarchy :)
